### PR TITLE
About us

### PR DESCRIPTION
**Pull request by TheCodeDestroyer**
First sample draft of `About Us`

### CODE REVIEW

**Comment by alexweber** [comment]
(•_•) ( •_•)>⌐■-■ (⌐■_■)

**Comment by TheCodeDestroyer** [comment]
@alexweber Should I put that in instead? But the before looks like boobs with double nipples...

**Comment by alexweber** [comment]
@TheCodeDestroyer hahahha I like just this for shades: (⌐■_■)

**Comment by TheCodeDestroyer** [comment]
@alexweber True :)

File: about-us.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+We are a group of pretty diverse people, who absolutely love and enjoy our calling,
+which is of course coding. We do it for our passion, hence the name.
+This group is formed by people with different areas of expertise,
+implying that our topics will be quite broad.
+We also love to be a part of the community, like taking part in open source projects,
+sharing our knowledge and ideas.
+
+Our goal is to produce content that will be interesting for you to read in a casual,
+maybe even goofy form of a way.
```
Comment (left-semi-join) [inline about-us.md]: "...produce informative content in interesting and easy to read form" - for example? I'm not sure I like "goofy" being used although I know what you mean and the idea is OK to me, but officially labelling it as goofy is maybe a bit too far? Also, "form of a way" sort of scratches it the wrong way for me.

Comment (TheCodeDestroyer) [inline about-us.md]: @left-semi-join Sure makes sense, I'll change it.

**Pull request merged**